voluptuous/schema_builder.py

```diff
@@ -1148,7 +1159,7 @@ class Required(Marker):
     {'key': []}
     """
 
-    def __init__(self, schema, msg=None, default=UNDEFINED, description=None):
+    def __init__(self, schema: dict, msg: typing.Optional[str] = None, default=UNDEFINED, description: typing.Optional[str] = None) -> None:
```

Comment:
Is this correct? `Required` can also take other scalar types e.g. the example in its docstring: `schema = Schema({Required('key'): str})`

Reply:
No I don't think it is. Want to send a PR?

Reply:
Ah, I've overlooked that one. Commit coming up